Sources/ArcGISToolkit/Components/Offline/OfflineMapViewModel.swift

```diff
@@ -41,6 +41,14 @@ class OfflineMapViewModel: ObservableObject {
         onlineMap.loadStatus == .loaded && onlineMap.offlineSettings == nil
     }
     
+    /// A Boolean value indicating whether there are downloaded map areas for the web map.
+    var hasDownloadedMapAreas: Bool? {
+        guard case.success(let preplannedModels) = preplannedMapModels,
+              let onDemandModels = onDemandMapModels else { return nil }
```

Comment:
onDemandModels is no longer optional. Please merge latest base branch into your dev branch.